app/src/main/java/org/schabi/newpipe/about/AboutActivity.java

```diff
@@ -92,16 +93,24 @@ protected void onCreate(final Bundle savedInstanceState) {
         Toolbar toolbar = findViewById(R.id.toolbar);
         setSupportActionBar(toolbar);
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
-        // Create the adapter that will return a fragment for each of the three
-        // primary sections of the activity.
-        mSectionsPagerAdapter = new SectionsPagerAdapter(getSupportFragmentManager());
 
         // Set up the ViewPager with the sections adapter.
         mViewPager = findViewById(R.id.container);
+
+        Configuration config = getResources().getConfiguration();
+        boolean isRTL = config.getLayoutDirection() == View.LAYOUT_DIRECTION_RTL;
```

Comment:
As this method is duplicated thrice, you should make it a static util method `boolean isRTL(Context context)` and then use `context.getResources()`.